normandy/settings.py

```diff
@@ -116,6 +124,11 @@ class Production(Base):
     """Settings for the production environment."""
     STATICFILES_STORAGE = 'whitenoise.django.GzipManifestStaticFilesStorage'
 
+    DEFAULT_FILE_STORAGE = 'storages.backends.s3boto.S3BotoStorage'
```

Comment:
[The docs say](http://django-storages.readthedocs.org/en/latest/backends/amazon-S3.html#usage) that the boto backend requires the `python-boto` package to be installed, and it doesn't do it automatically.